cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -31,10 +33,71 @@
 import static com.powsybl.cgmes.conversion.Conversion.Config.StateProfile.SSH;
 
 /**
+ * TwoWindingsTransformer Interpretation
+ * <p>
+ * Ratio and Phase Interpretation (Xfmr2RatioPhaseInterpretationAlternative) <br>
+ * END1. All tapChangers (ratioTapChanger and phaseTapChanger) are considered at end1 (before transmission impedance) <br>
+ * END2. All tapChangers (ratioTapChanger and phaseTapChanger) are considered at end2 (after transmission impedance) <br>
+ * END1_END2. TapChangers (ratioTapChanger and phaseTapChanger) are considered at the end where they are defined in Cgmes <br>
+ * X. If x1 == 0 all tapChangers (ratioTapChanger and phaseTapChanger) are considered at the end1 otherwise they are considered at end2
+ * <p>
+ * Phase Angle Negate. All phaseTapChanger angles are considered -angle.
+ * <p>
+ * Shunt Admittance Interpretation (Xfmr2ShuntInterpretationAlternative) <br>
+ * END1. All shunt admittances to ground (g, b) at end1 (before transmission impedance) <br>
+ * END2. All shunt admittances to ground (g, b) at end2 (after transmission impedance) <br>
+ * END1_END2. Shunt admittances to ground (g, b) at the end where they are defined in Cgmes model <br>
+ * SPLIT. Split shunt admittances to ground (g, b) between end1 and end2. <br>
+ * <p>
+ * Structural Ratio (Xfmr2StructuralRatioInterpretationAlternative) <br>
+ * END1. Structural ratio always at end1 (before transmission impedance) <br>
+ * END2. Structural ratio always at end2 (after transmission impedance) <br>
+ * X. If x1 == 0 structural ratio at end1, otherwise at end2
+ * <p>
+ * ThreeWindingsTransformer Interpretation.
+ * <p>
+ * Ratio and Phase Interpretation.  If it is true all tapChangers (ratioTapChanger and phaseTapChanger) at the network side, otherwise at the star bus side
```

Comment:
Can you explain us why the default interpretation is RTC and PTC localized at network side ? Can you reference this information in IEC TS 61970-600-2:2017 ?

Reply:
According with the Cgmes Reference IEC 61970-452:2017 page 9. Section “4.3 Transformer modeling” the `structruralRatio` in three windings transformers is located at the star bus side. 

Also, the `tapChangers` are always located at the network side in three windings transformers. 
This convention  is also applied in the current threeWindingsTransformer conversion. (See the method `rtcAtSide1()` in `RatioTapChangerConversion.java` module. 

We use the same convention in the new Cgmes conversion.

Reply:
Thanks for your detailed answer. I can well understand that structural ratio is at star bus side. But it is not so clear for me that tap changers are always located at network side. In the CIM-CGMES 3 windings transformer modeling, what does it mean when the structural ratio and the RTC/PTC are not located in the same place in the leg ? Where can I see that PTC/RTC are located at network side ?

Reply:
Theoretically it is possible to have structural ratio in one side and the RTC in the other.  I do not know what physically means.
The RTC in threeWindingsTransformer was always considered in the network side in the current cgmes conversion and it was confirmed by the model interpretation tool. We only get a zero balance considering RTC in network side.